File: crudAngularEWebApi/AppAngular/src/app/Pessoa.ts
export class Pessoa{
    UserID: Number
    Name: String
    LastName: String
    Age: Number

    constructor(userId: Number, name: String, LastName: String, age: Number){
        this.UserID = userId
        this.Name = name
        this.LastName = LastName
        this.Age = age
    }
}

File: crudAngularEWebApi/CrudAPI/Models/Pessoa.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CrudAPI.Models
{
    public class Pessoa
    {
        [Key]
        public int UserID { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public int Age { get; set;}
    }
}